### PR TITLE
Remove last offscreen focused item

### PR DESCRIPTION
**Pull request by keanulee**

### CODE REVIEW

File: test/focus.html [hunk 1]
```diff
@@ -107,17 +107,21 @@
...
-          done();
+      flush(function() {
+        list.scroll(0, getFirstItemFromList(list).offsetHeight * list._physicalCount);
+       
+        requestAnimationFrame(function() {
+          setTimeout(function() {
+            var lastOffscreenFocusedItem = list._offscreenFocusedItem;
+            var firstItem = getFirstItemFromList(list);
+            firstItem.parentNode.focus();
+
```
Comment (blasten) [inline test/focus.html]: I think it now needs `Polymer.dom.flush()` here.

Comment (keanulee) [inline test/focus.html]: Will give that a try

**Comment by keanulee** [comment]
Because of the way focus is handled in Chrome, the test will always fail if the browser window is not focused (but passes when the window is focused). This behavior can be replicated locally by reloading the test page the switching to DevTools or a different window before that test runs.

I've added a conditional that skips this test if the window doesn't have focus, but this seems like a pretty dirty hack.

**Comment by blasten** [comment]
LGTM :)

**Pull request merged**